src/utl/README.md

```diff
@@ -0,0 +1,63 @@
+# Utilities
+
+The utility module houses of the manpage command in OpenROAD,
+available from the `man` command.
+
+## Commands
+
+```{note}
+- Parameters in square brackets `[-param param]` are optional.
+- Parameters without square brackets `-param2 param2` are required.
+```
+
+### Man
+
+This can be used for a range of commands in different levels as follows:
+- `man1`: Top-level openroad command (e.g. `man openroad`)
+- `man2`: Individual module commands (e.g. `man clock_tree_synthesis`)
+- `man3`: Info, error, warning messages (e.g. `man CTS-0001`)
```

Comment:
Confusing as it sounds like there are man1/man2/man3 commands rather than three usages of the one man command.

Reply:
Changed to "Level 1/2/3".